src/Plugin/OgDeleteOrphans/Cron.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Drupal\og\Plugin\OgDeleteOrphans;
+
+use Drupal\og\OgDeleteOrphansBase;
+
+/**
+ * Performs a cron deletion of orphans.
+ *
+ * @OgDeleteOrphans(
+ *  id = "cron",
+ *  label = @Translation("Cron", context = "OgDeleteOrphans"),
+ *  description = @Translation("The deletion is done in the background during cron. Best overall solution but requires cron to run regularly."),
+ *  weight = 3
+ * )
+ */
+class Cron extends OgDeleteOrphansBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function process() {
```

Comment:
this and the method below confuses me a bit - can you please clarify how it's working + why we use a different queue.